examples_utils/benchmarks/logging_utils.py

```diff
@@ -76,45 +76,47 @@ def print_benchmark_summary(results: dict):
         print(f"================ {failed} failed, {passed} passed ===============")
 
 
-def get_checkpoint_dir(cmd: list):
-    """Get the root dir where checkpoints are storied for a model.
+def get_latest_checkpoint_path(benchmark_path: str, variant_command: list) -> Path:
+    """Get the path to the latest available checkpoint for a model.
 
     Args:
+        benchmark_path (str): The path to the benchmarks.yml file
         cmd (str): The command used for this model run (benchmark)
```

Comment:
This has been changed to variant command and so needs to be updated

Reply:
Thanks, updated